### PR TITLE
Prepare packages for 4.0.0-preview release.

### PR DESCRIPTION
**Pull request by jimmyca15**

### CODE REVIEW

**Review by zhiyuanliang-ms — APPROVED** [review]

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
   <!-- Official Version -->
   <PropertyGroup>
-    <MajorVersion>3</MajorVersion>
-    <MinorVersion>1</MinorVersion>
-    <PatchVersion>1</PatchVersion>
+    <MajorVersion>4</MajorVersion>
+    <MinorVersion>0</MinorVersion>
+    <PatchVersion>0</PatchVersion>
```
Comment (zhiyuanliang-ms) [inline src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:8]: Should there be something like  <PreviewVersion>-preview</PreviewVersion>?
 I noticed that we have this in 3.0.0-preview.

File: src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/Microsoft.FeatureManagement.Telemetry.ApplicationInsights.csproj [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>false</DelaySign>
+    <AssemblyOriginatorKeyFile>..\..\build\Microsoft.FeatureManagement.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>Microsoft.FeatureManagement.AspNetCore provides a way to develop and expose application functionality based on features. Many applications have special requirements when a new feature is developed such as when the feature should be enabled and under what conditions. This library provides a way to define these relationships, and also integrates into common ASP.NET Core code patterns to make exposing these features possible.</Description>
```
Comment (zhiyuanliang-ms) [inline src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/Microsoft.FeatureManagement.Telemetry.ApplicationInsights.csproj:22]: The description looks like incorrect here.

Comment (jimmyca15) [reply src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/Microsoft.FeatureManagement.Telemetry.ApplicationInsights.csproj]: Updated. Thanks.

File: src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj (hunk 1 above)
Comment (jimmyca15) [reply src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj:8]: Updated. Thanks.

**Pull request merged**